### ISSUE TITLE
Which datasets are used to train Semantic-SAM?

### ISSUE THREAD

**Issue by yucornetto** [UX-Decoder/Semantic-SAM#68]
Thanks a lot for the great work! I encountered some questions when reading your paper:

In section 2 "Data Unification: Semantics and Granularity", it is claimed that 7 datasets are included so that semantic-SAM can segment and classify at "multi-level semantics".
<img width="1159" alt="image" src="https://github.com/UX-Decoder/Semantic-SAM/assets/24356755/b03ea4c9-b818-4ef1-9041-f2699108d7f6">

However, in the reported results in your arxiv paper, I only see settings of SA-1B only (Table 5, 6, 7, 9), SA-1B + COCO (Table 3, 8), SA-1B + Pascal Part (Table 4)

Where are Object365, ADE20K, PACO-LVIS, and Part-ImageNet used?

Thanks for your time and the great work!

**Comment by FengLi-ust** [comment]
Yes, the performance of the large model with all datasets will be released in our updated version.
